SS2-Project/Assets/Starstorm2/Modules/Pickups/Items/Tier1/Needles.cs

```diff
@@ -78,5 +93,32 @@ public void OnIncomingDamageOther(HealthComponent self, DamageInfo damageInfo)
                 }
             }
         }
+
+        public sealed class Needle : BaseBuffBehaviour, RoR2.IOnIncomingDamageServerReceiver
+        {
+            [BuffDefAssociation]
+            public static BuffDef GetBuffDef() => SS2Content.Buffs.BuffNeedle;
+
+            private void Start()
+            {
+                if (CharacterBody.healthComponent)
+                    HG.ArrayUtils.ArrayAppend(ref CharacterBody.healthComponent.onIncomingDamageReceivers, this);
+            }
+            public void OnIncomingDamageServer(DamageInfo info)
+            {
+                info.crit = true;
+            }
+
+            private void OnDestroy()
```

Comment:
Fossil from kevin days, can be removed

Reply:
Youre talking about the buff?

Reply:
nvm yeah the NeedleBuff, it isn't referenced anywhere it seems